Winch/Serialization/Item/FishItemDataConverter.cs

```diff
@@ -2,26 +2,26 @@
 using Newtonsoft.Json.Linq;
 using UnityEngine;
 
-// ReSharper disable HeapView.BoxingAllocation
-
 namespace Winch.Serialization.Item;
 
 public class FishItemDataConverter : HarvestableItemDataConverter
 {
     private readonly Dictionary<string, FieldDefinition> _definitions = new()
     {
+        { "itemType", new(ItemType.GENERAL, null) },
+        { "itemSubtype", new(ItemSubtype.FISH, null) },
         { "minSizeCentimeters", new( 0f, o => float.Parse(o.ToString())) },
         { "maxSizeCentimeters", new( 0f, o => float.Parse(o.ToString())) },
         { "aberrations", new( new List<FishItemData>(), null) },
-        { "isAberration", new( false, null) },
+        { "isAberration", new( false, o => bool.Parse(o.ToString())) },
         { "nonAberrationParent", new( null, null) },
```

Comment:
Not quite sure how we'd go about assigning the parent here before the game even loads the other items

Reply:
I'm still in the camp of lazy object creation. We can adapt the converters to deal with string ids and have actual game object instantiated at later point of game loading